### PR TITLE
Testing why travis builds aren't being triggered

### PR DESCRIPTION
**Pull request by AbdealiLoKo**

### CODE REVIEW

File: file_metadata/image/svg_file.py [hunk 1]
```diff
@@ -35,3 +35,15 @@ def fetch(self, key=''):
                 return name
 
         return super(SVGFile, self).fetch(key)
+
+    def analyze_file_format(self):
+        """
+        Simply add a metadata mentioning this is a valid SVG file. This is
+        useful because SVG cannot be simply detected by MimeType as it's not
+        standardized yet.
```
Comment (drtrigon) [inline file_metadata/image/svg_file.py:43]: What about using `from py_w3c.validators.html.validator import HTMLValidator, ValidationFault` to check SVG in general and for validity?

Comment (AbdealiLoKo) [inline file_metadata/image/svg_file.py:43]: I do have a SVG checking method - not using py_w3c, but an equivalent one. That's how this class is decided upon by GenericFile

**Comment by drtrigon** [comment]
From looking at `is_svg()` in https://github.com/pywikibot-catfiles/file-metadata/blob/master/file_metadata/mixins.py#L98 I do not understand how you figure out invalid SVGs (according to W3C). That would be useful to categorize by filling templates like https://commons.wikimedia.org/wiki/Category:SVG_created_with_..._templates

But this is not related to this issue.

**Comment by AbdealiLoKo** [comment]
Agree, not related, but answering anyway ...
The algo I used was a quick and dirty copy from http://stackoverflow.com/questions/15136264/how-can-i-say-a-file-is-svg-without-using-a-magic-number which worked for the files I tested on commons. It was only intended to check if a file was SVG or not.

At that time I did not like py_w3c because the latest version was 0.1.2 which was created in 2014 but it seems 0.2.0 was released just 15 days ago - which re-affirms my faith in the library :)
PS: Python3 is still not supported in py_w3c 0.2.0 :(

**Comment by drtrigon** [comment]
15 days ago... I feeled it... ;)

They mention py3.4 support but I could not find a travis yet.

**Pull request closed without merging**